sycl/test/fpga_tests/fpga_queue.cpp

```diff
@@ -42,9 +43,10 @@ int getExpectedQueueNumber(cl_device_id device_id, int default_value) {
                                  sizeof(reportedProps),
                                  &reportedProps,
                                  NULL);
-   assert(CL_SUCCESS == iRet && "Failed to obtain queue info from ocl device");
+   CHECK(CL_SUCCESS == iRet && "Failed to obtain queue info from ocl device");
```

Comment:
Why only part of the asserts are replaced in this file?

Reply:
Thank you for catching this. Breaking a larger PR down into smaller ones I might have lost some changes (or I was sloppy in the first place). I will look into it.